app/kudos/templates/share.html

```diff
@@ -0,0 +1,142 @@
+{% load i18n static %}
+{% load kudos_extras %}
+<style>
+  .modal {
+    display: none;
+    position: fixed;
+    z-index: 1025;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0,0.4);
+    max-width: 100%;
+    overflow: scroll;
```

Comment:
maybe better to just `overflow-y: auto; ` to avoid unwanted scrolls or scrolling when is not needed